Mutuba/Ride-My-Way2
# #158564443 Have API endpoints working

### What does the PR do?

Implements the following endpoints:
- POST/api/v1/auth/register
- POST/api/v1/auth/login
- POST/api/v1/auth/change_password
- POST/api/v1/auth/logout
- POST/api/v1/rides
- GET/api/v1/rides
- GET/api/v1/rides/ride_id
- POST/api/v1/requests
- GET/api/v1/requests
- DELETE/api/v1/rides/ride_id
- PUT/api/v1/rides/ride_id

## How to manually test
- Clone the repo from github. Change directory into Ride-My-Way2 and run python run.py to start the server.
- Open postman and test all the above endpoints to see the results.
- Remember after registering, login gives you an access jwt token to authenticate you for resources that need token.

## Relevant pivotal tracker stories
- #158564443
